### DISCUSSION TITLE
Newbie here please help

### DISCUSSION THREAD

**Discussion by ivanvach** [parnic/node-screenlogic#63] [General]
I'm not a programmer and I can't use the libraries as they are.
Is there any API that I can use to interact with my Pentair Controller.
I do have the SceeenLogic adapter as well installed and working.

https://www.poolweb.com/products/interface-pc-itc15-intellitouch

I found this implementation of @parnic library

mikemucc/screenlogic-api

and I managed to install and run it, however it doesn't work for me and there is no way to contact the author or get any kind of support.

Please help if you can. Appreciated. 

Best regards,

Ivan

**Comment by parnic** [comment]
This is only a library, to be used by other developers to add Pentair support to their apps. I can't help with @mikemucc 's application.

**Comment by ivanvach** [comment]
@parnic 

Thanks. I'm trying to reach out to @mikemucc with no luck so far. That's why I'm trying other avenues as well.

Best,
Ivan
